modules/microshift-stopping-lvms-csi-snapshot.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-stopping-lvms-csi-snapshot_{context}"]
+== Stopping deployments that run CSI implementations
+
```

Comment:
My understanding is that removing the CSI makes the cluster no longer compliant with CNCF, so we need an IMPORTANT admonition in here.

Reply:
MicroShift deploys dynamically provisioned storage by default, so it meets CNCF standards. I understand the importance of being CNCF compliant from a dev perspective. It seems weird to me that a customer would care if their particular deployment met this requirement as well.

Reply:
resolving as discussed